server/src/main/java/org/opentosca/toscana/plugins/cloudformation/CloudFormationScriptCreator.java

```diff
@@ -0,0 +1,55 @@
+package org.opentosca.toscana.plugins.cloudformation;
+
+import java.io.IOException;
+
+import org.opentosca.toscana.core.plugin.PluginFileAccess;
+import org.opentosca.toscana.plugins.scripts.BashScript;
+import org.opentosca.toscana.plugins.scripts.EnvironmentCheck;
+
+/**
+ * Class for creating scripts needed for AWS CloudFormation deployment.
+ */
+public class CloudFormationScriptCreator {
+    private static final String CLI_COMMAND_CREATESTACK = "aws cloudformation deploy ";
+    private static final String CLI_PARAM_STACKNAME = "--stack-name ";
+    private static final String CLI_PARAM_TEMPLATEFILE = "--template-file ";
+    //TODO check if PARAMOVERRIDES are needed and get said parameters
```

Comment:
I removed this from the command for now, since `--parameter-overrides` without any paramters won't start the depoyment.
I'm guessing we'll have to get this from the parameters in cfnModule?

Reply:
I added an issue for this: #413